lib/networking/metallb_l2advertisement.yaml

```diff
@@ -12,8 +12,7 @@ spec:
   ipAddressPools:
   - ctlplane
   interfaces:
-  # Change this interface to the appropriate OpenStack NIC for your environment
-  - enp7s0 # CHANGEME
+  - _repalced_
```

Comment:
typo? here and below

Reply:
This is not a typo. 

Values like `enp7s0` should not be in the `lib` directory. They go in the values.yaml file under `examples`. So where those values were in the `lib` directory, I put a dummy value like `_replaced_`. Then when kustomize is run, the rendered output has the value and the dummy value is replaced.

Reply:
yes; i meant there's a real typo in here :)

Reply:
Oh! I didn't see it because I read `replaced` too quickly. You meant "you spelled 'replaced' wrong".
```
-  - _repalced_
+  - _replaced_
```